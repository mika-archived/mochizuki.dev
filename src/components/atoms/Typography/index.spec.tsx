import "@testing-library/jest-dom/extend-expect";
import "jest-styled-components";

import React from "react";
import { cleanup, render } from "@testing-library/react";

import { Small, Normal, Large } from ".";

afterEach(cleanup);

test("does not have changes from snapshot in Small", () => {
  const { asFragment } = render(<Small>Small</Small>);

  expect(asFragment()).toMatchSnapshot();
});

test("does not have changes from snapshot in Normal", () => {
  const { asFragment } = render(<Normal>Normal</Normal>);

  expect(asFragment()).toMatchSnapshot();
});

test("does not have changes from snapshot in Large", () => {
  const { asFragment } = render(<Large>Large</Large>);

  expect(asFragment()).toMatchSnapshot();
});
