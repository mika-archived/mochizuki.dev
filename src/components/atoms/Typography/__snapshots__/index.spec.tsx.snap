// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`does not have changes from snapshot in Large 1`] = `
<DocumentFragment>
  .c0 {
  margin: 0;
  font-family: "游ゴシック体","Yu Gothic",YuGothic,"ヒラギノ角ゴシック Pro","Hiragino Kaku Gothic Pro",メイリオ,Meiryo,sans-serif;
  font-size: 18px;
}

<p
    class="sc-AxjAm c0"
  >
    Large
  </p>
</DocumentFragment>
`;

exports[`does not have changes from snapshot in Normal 1`] = `
<DocumentFragment>
  .c0 {
  margin: 0;
  font-family: "游ゴシック体","Yu Gothic",YuGothic,"ヒラギノ角ゴシック Pro","Hiragino Kaku Gothic Pro",メイリオ,Meiryo,sans-serif;
  font-size: 16px;
}

<p
    class="sc-AxjAm c0"
  >
    Normal
  </p>
</DocumentFragment>
`;

exports[`does not have changes from snapshot in Small 1`] = `
<DocumentFragment>
  .c0 {
  margin: 0;
  font-family: "游ゴシック体","Yu Gothic",YuGothic,"ヒラギノ角ゴシック Pro","Hiragino Kaku Gothic Pro",メイリオ,Meiryo,sans-serif;
  font-size: 14px;
}

<p
    class="sc-AxjAm c0"
  >
    Small
  </p>
</DocumentFragment>
`;
